src/services/TableService.js

```diff
@@ -16,9 +16,10 @@ function TableService($q, $http, URLService, UtilsService, CacheService, _, Elem
      *
      * @param {string} mmsEid SysML ID of a table instance
      */
-    var readTableCols = function(mmsEid, ws, version) {
+    var readTableCols = function(reqOb, ws, version) {
       var deferred = $q.defer();
-      ElementService.getElement(mmsEid, false, ws, version)
+      //ElementService.getElement(mmsEid, false, ws, version)
+      ElementService.getElement(reqOb, 1, true)
```

Comment:
last argument can be false or just leave off, if false it'll get from the cache if availalbe